app/views/admin/users/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+
+<%= form_for @user, :url => {:action => "update", :id => @user.id } do |f| %>
```

Comment:
Hm you shouldn't need the `:url` hash should you? Those should be the defaults for an edit form.

Reply:
There not. This form is for editing a user, so it redirects to user associated things not admin associated things.